### DISCUSSION TITLE
Is there a demand to include JUnit XML export in the core?

### DISCUSSION THREAD

**Discussion by atty303** [com-lihaoyi/mill#2700] [Ideas]
If #1779 is not supported, test failure cannot be reported. Therefore, the XML export must be incorporated into TestModule's testTask. Here we support it by overriding testTask, but it might be useful if it is supported in the Mill core.
I can PR it, but please let me know what you think.

**Comment by lefou** [comment]
I'd definitely like that feature.

**Reply by lefou** [reply]
@atty303 If preparing a PR isn't possible to you, maybe you can share the code you already have. There is definitely demand in this feature!

**Reply by romain-gilles-ultra** [reply]
Hi,
I'm realy looking forward to a solution to get JUnit XML reports at the end of my test execution.
I developed something to do so that I can share if you want.

**Reply by atty303** [reply]
@lefou Sorry, since I was no longer using it in my own project, I had neglected it. I have merged the code into the Mill main repository and created a pull request #3099.

**Reply by lefou** [reply]
Thank you, @atty303 . Your PR is merged.
